Tests/Runtime/MobiledgeX_RuntimeTests.cs

```diff
@@ -259,11 +259,20 @@ public async Task<string> GetUrlHelper(MobiledgeXIntegration mxi, string proto,
                 throw new FindCloudletException("FindCloudlet Failed");
             }
 
-            AppPort appPort = proto switch
+            AppPort appPort;
+            switch(proto)
             {
-                "udp" => mxi.GetAppPort(LProto.L_PROTO_UDP, port),
-                _ => mxi.GetAppPort(LProto.L_PROTO_TCP, port),
-            };
+                case "udp":
+                    appPort = mxi.GetAppPort(LProto.L_PROTO_UDP, port);
+                    break;
+                default:
+                case "ws":
+                case "wss":
+                case "http":
+                case "https":
+                    appPort = mxi.GetAppPort(LProto.L_PROTO_TCP, port);
+                    break;
```

Comment:
nit: would prefer default to be at the bottom and everything else falls through into that